service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/gcsbucket/ControlledGcsBucketHandler.java

```diff
@@ -28,4 +42,9 @@ public WsmResource makeResourceFromDb(DbResource dbResource) {
             .build();
     return resource;
   }
+
+  public String generateCloudName(UUID workspaceUuid, String bucketName) {
```

Comment:
@nullable workspaceUuid

and check null

Reply:
the other option is to have a generateCloudName and a generateCloudNameWithWorkspaceId then you don't need to do nullcheck since it won't be nullable.